base_classes/NXdata.nxdl.xml

```diff
@@ -60,176 +60,181 @@
 			does not describe how the data is to be plotted or even the dimensionality of the plot.
 			https://www.nexusformat.org/NIAC2018Minutes.html#nxdata-plottype--attribute
 
+		**Usage:**
+
+		.. admonition:: Plot data versus x axis
+
+			.. code-block::
+
+				data:NXdata
+				  @signal = "data"
```

Comment:
I did not know this. Not specified in the definition. I'll add it.

Reply:
I guess when `@axes` is missing, we use `[".", ".", ...]` (as many dots as rank)?

Reply:
If `@signal` is missing, there is no rule that says `"data"` is required as a field name.  In this case and multiple fields, selection of the _signal_ field for the default plot is not guaranteed.

Reply:
I thought `NXdata` evolved from a fixed signal field (usually called `data`, see e.g. https://github.com/nexusformat/definitions/blob/04bfe4ccbbfd3cb715d1a1f42990a2d18fa5ed72/base_classes/NXdata.nxdl.xml#L152) with an (optional?) attribute `@signal=1` so to be backwardly compatible, a modern parser could assume the group attribute `signal` if missing is `"data"`.

Reply:
I think that @prjemian is correct. I just scanned through the minutes of all the NIAC meetings, and I can't find any reference to a default name for the `signal`. I believe that the current (non-deprecated) method of defining signals and axes requires both `signal` and `axes` attributes to be defined at group level.

Reply:
It was found that assigning these links to fields would break when the fields were linked (local or external) into other groups.  Such field attributes could produce a group with multiple field `@signal` attributes.

Reply:
Perhaps we need to make these attributes `required` then.

Reply:
Could bring that up again for a NIAC vote.  Not until then, since NIAC voted the other way.  

Consider a simple use case (for not having this attribute).  User wants to save a 2-D image, only the 2-D image.  NeXus file consists of `/entry:NXentry/data:NXdata/image` where `image` is the 2-D image.  We've gotten strong pushback on requiring `@signal` in this use case.